addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/LdapInjectionScanRule.java

```diff
@@ -222,7 +224,25 @@ public void scan(HttpMessage originalmsg, String paramname, String paramvalue) {
             // get the response for the "original" unmodified request
             // this fixes what seems to be a bug in the Zap core, where the request response is not
             // actually available at this point via "originalmsg"
-            sendAndReceive(originalmsg);
```

Comment:
I'm not sure what issue this is addressing. Instead of sending originalmsg (which is actually the message passed into the scan method) should this simply leverage `getBaseMsg()` and not resend the msg being passed in?

Reply:
This was added to make sure that the message had a response (e.g. messages of temporary nodes). This is no longer needed as core ensures that the messages have a response.

Reply:
And to answer your question, yes, it can use the base message.